### PR TITLE
Fix a typo in 6.0 release notes

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Fix a typo in 6.0 release notes and update collation results
Close https://github.com/pingcap/docs-cn/issues/9266 https://github.com/pingcap/docs-cn/issues/9265

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-show-collation.md [hunk 1]
```diff
@@ -57,13 +57,17 @@ SHOW COLLATION;
 +--------------------+---------+------+---------+----------+---------+
 | ascii_bin          | ascii   |   65 | Yes     | Yes      |       1 |
 | binary             | binary  |   63 | Yes     | Yes      |       1 |
+| gbk_bin            | gbk     |   87 |         | Yes      |       1 |
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-collation.md:60]: @wjhuang2016 PTAL. Thanks!

Comment (wjhuang2016) [reply sql-statements/sql-statement-show-collation.md:60]: LGTM

File: releases/release-6.0.0-dmr.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ v6.0.0 是 DMR 版本，版本名称为 6.0.0-DMR。
         - 逻辑函数： `IS`，`IS NOT`
         - 字符串函数：`REGEXP()`，`NOT REGEXP()`
         - 数学函数：`GREATEST(int/real)`，`LEAST(int/real)`
-        - 日期函数：`DAYOFNAME()`，`DAYOFMONTH()`，`DAYOFWEEK()`，`DAYOFYEAR()`，`LAST_DAY()`，`MONTHNAME()`
+        - 日期函数：`DAYNAME()`，`DAYOFMONTH()`，`DAYOFWEEK()`，`DAYOFYEAR()`，`LAST_DAY()`，`MONTHNAME()`
```
Comment (windtalker) [inline releases/release-6.0.0-dmr.md:127]: LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**